src/randomize_music.js

```diff
@@ -14,95 +14,150 @@
     util = require('./util')
   }
 
-  const music = {
-    LOST_PAINTING: [
+  const musicBySong = [
```

Comment:
Converting the `MUSIC` object to an array and unmapping `music` here is a significant step backwards for developer friendliness. It should be entirely possible to apply your work without removing the map. If you want an array of an object's values that preserves its declaration order, you can simply use [`Object.values`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values).

Reply:
I don't see how it makes it "less friendly". It's simpler to work with this way, and this is the only piece of code that uses this data anyway. I could add comments, like I did for the constants already, but making it an object instead of an array seems to just add an extra step to do anything with it.